wireless_modules/middle_module/README.md

```diff
@@ -0,0 +1,62 @@
+# Data Acquisition System - Middle wireless module
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors)
+
+The Middle Wireless Module allows us to collect and send data from x sensors attached to the bike. The middle wireless module consists of a micro controller (ESP32 currently) which gathers the data from the on board sensors and sends them using MQTT (using the internet).
+
+This repository contains all the code related to Monash Human Power's middle wireless module - part of their Data Acquisition System (DAS).
+
+## Getting Started
+
+1. Press clone on the root folder and use it to clone the entire repository on the command line.
+2. Create a config.py file in the same directory as the contents of this folder (refer below).
+3. The config.py file should hold the following variables (ensure the name matches):
+- ESSID : Should be assigned to a string containing the SSID of the network to connect to
```

Comment:
Monospace for code variables like this:

- `ESSID` : Should be assigned to a string containing the SSID of the network to connect to